### DISCUSSION TITLE
"Améliorations potentielles" - Détectées lors du webinaire du 05-11

### DISCUSSION THREAD

**Discussion by JeremieSeguin** [MTES-MCT/sparte#717] [General]
Lors de l'animation d'un webinaire, je peux être amener à trouver des petites coquilles sur la plateforme (qu'on décidera ou non de les corriger)

J'ai créé cette discussion pour remonter les suggestions/remarques sorties du webinaire du 05-11

**Comment by JeremieSeguin** [comment]
Diagnostic > Enjeux environnementaux > Cadre réglementaire 

L'apparition sur la droite de ce cadre réglementaire ne permet pas de lire les éléments de la matrice de l'artificialisation 

<img width="1527" alt="image" src="https://github.com/user-attachments/assets/bc336c32-4d95-4704-83e4-70d8b86d2e98">

**Comment by JeremieSeguin** [comment]
Dans une carto OCS GE, les éléments de la légende ne s'affiche pas entièrement rendant celle ci difficile à la compréhension 

Exp : Zone à matériaux ... 

<img width="764" alt="image (1)" src="https://github.com/user-attachments/assets/09ef6595-45ca-4eaa-951e-28d6fed9b235">

**Comment by JeremieSeguin** [comment]
Dans l'explorateur des zonages d'urbanismes les surfaces des objets et des zones ne me semble pas compréhensible. 

Dans l'exemple ci-dessous je comprends que je sélectionne un objet d'une surface de 3 709,86 ha alors que celui ci semblerait être à l'intérieur d'une zone urbaine de 20,3 ha

![Surface des objets OCS GE](https://github.com/user-attachments/assets/f98f76e3-f24d-4b7e-b6a4-74f044be6e44)

**Reply by alexisig** [reply]
bien vu merci ! je transforme ce point en issue

**Reply by JeremieSeguin** [reply]
Top, merci
